clearcut_detection_backend/test/scripts/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6
```

Comment:
you using another dockerfile in docker-compose (test.Dockerfile) why do We need this one?